frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -31,6 +31,15 @@ pods:
             vip:
```

Comment:
We should also make the broker plain text port only be advertised if TLS is enabled and plaintext comms are still allowed.